release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1727,10 +1732,55 @@ If these values are not defined in a failure domain configuration, for instance
+
+* Previously, release images were signed using a SHA-1 key. On {op-system-base} 9 FIPS STIG-compliant machines, verification of release signatures using the old SHA-1 key failed due to security restrictions on weak keys. With this release, release images are signed using a new SHA-256 trusted key so that the release signatures no longer fail. (link:https://issues.redhat.com/browse/OCPBUGS-48314[*OCPBUGS-48314*])
+
+* Previously, when running oc-mirror v2 Technology Preview in mirror-to-disk mode within an disconnected environment, the tool attempted to download `graph.tar.gz` from `api.openshift.com`, causing the mirroring process to fail.
++
+With this fix, when the `UPDATE_URL_OVERRIDE` environment variable is set, oc-mirror first checks for the graph image in the oc-mirror cache. If the image is not found, oc-mirror skips it without failing, ensuring successful mirroring in air-gapped environments. (link:https://issues.redhat.com/browse/OCPBUGS-38037[*OCPBUGS-38037*])
+
+* Previously, oc-mirror v2 Technology Preview did not support images referenced by both `tag` and `digest`. These images were excluded from the archive during the mirror-to-disk workflow, causing failures when mirroring from the archive to a disconnected mirror registry. With this fix, oc-mirror v2 includes these images in the archive by pulling the source image by `digest` while retaining the `tag` for reference. A warning message is displayed to inform users of this behavior.  (link:https://issues.redhat.com/browse/OCPBUGS-37867[*OCPBUGS-37867*])
+
+* Previously, when using the --force-cache-delete flag to delete images from a remote registry, the deletion process did not work as expected. With this update, the issue has been resolved, ensuring that images are deleted properly when the flag is used. (link:https://issues.redhat.com/browse/OCPBUGS-47690[*OCPBUGS-47690*])
+
+* Previously, oc-mirror plugin v2 could not delete the graph image when mirrored using a partially disconnected mirroring workflow (mirror-to-mirror). With this update, graph images can now be deleted regardless of the mirroring workflow used. (link:https://issues.redhat.com/browse/OCPBUGS-46145[*OCPBUGS-46145*])
+
+* Previously, if the same image was used by multiple OpenShift release components, oc-mirror plugin v2 attempted to delete the image multiple times, failing after the first attempt. This issue has been resolved by ensuring oc-mirror plugin v2 generates a list of unique images during the delete `--generate` phase. (link:https://issues.redhat.com/browse/OCPBUGS-45299[*OCPBUGS-45299*])
+
+* Previously, mirroring a release that used a digest instead of a tag did not work as expected. This issue has been fixed, ensuring successful mirroring for releases that use digests. (link:https://issues.redhat.com/browse/OCPBUGS-45249[*OCPBUGS-45249*])
```

Comment:
Hm, is this bug a clone of https://github.com/openshift/openshift-docs/pull/88938/files#diff-de8385a4382941faf5716af81fb130e1c918992441901815806051dbf153b083R1747 and a duplicate?